satpy/readers/ahi_l1b_gridded_bin.py

```diff
@@ -142,12 +142,12 @@ def _calibrate(self, data):
     @staticmethod
     def _download_luts(file_name):
         """Download LUTs from remote server."""
-        import requests
+        import urllib
         import shutil
         # Set up an connection and download
-        response = requests.get(AHI_REMOTE_LUTS, stream=True)
-        with open(file_name, 'wb') as out_file:
-            shutil.copyfileobj(response.raw, out_file)
+        with urllib.request.urlopen(AHI_REMOTE_LUTS) as response:
```

Comment:
```suggestion
        with urllib.request.urlopen(AHI_REMOTE_LUTS) as response:  # nosec
```

The url is not a variable, so we can skip the security check here I think.